pages/index.tsx

```diff
@@ -1,7 +1,5 @@
 import Head from "next/head";
-import Image from "next/image";
-import styles from "@/styles/Home.module.css";
-import {textFont, titleFont} from "./_app";
+import Hero from "@/components/hero";
```

Comment:
All components JSX/TSX files need to be capitalized. Use "Hero.tsx" instead of "hero.tsx".

Reply:
![unknown](https://user-images.githubusercontent.com/73293255/217449830-69003002-3b15-44ea-90dd-bffb65f9ca41.png)